mithril-install.sh

```diff
@@ -142,5 +142,15 @@ tar -xz -f "$ASSETS_FILE_PATH" -C "$INSTALL_PATH" "$NODE"
 chmod +x "$INSTALL_PATH/$NODE"
 rm -f "$ASSETS_FILE_PATH"
 
+if ! ERROR_OUTPUT=$("$INSTALL_PATH/$NODE" --version 2>&1); then
+  {
+    echo "Failed to run '$NODE --version'."
+    echo "Error details:"
+    echo "$ERROR_OUTPUT"
+    echo "The installation completed, but the binary could not be executed."
+  } >&2
+  exit 1
+fi
+
```

Comment:
There is a `set -e` which should stop the script on error. 

I think that it'd probably be better to split the computation of the version to have the error caught to avoid running it the command twice:
- create a variable with the command `VERSION_OUTPUT=$("$INSTALL_PATH/$NODE" --version)`
- use this variable to compute the version `VERSION=$(echo $VERSION_OUTPUT | awk '{print $NF}')`